maintenance/wikia/cronjobs/AutoFollow.php

```diff
@@ -2,24 +2,43 @@
 
 require_once( dirname(__FILE__) . '/../../commandLine.inc' );
 
-$alreadyWatchedKey = 'autowatched-already';
+global $wgCityId;
+$contentLanguage = WikiFactory::getVarValueByName( 'wgLanguageCode', $wgCityId );
 
 $dbr = wfGetDB( DB_SLAVE, array(), $wgExternalSharedDB );
-
 $range = date( 'Ymd', strtotime( '-30 days' ) ) . '000000';
-
 $res = $dbr->select( '`user`', 'user_id', array( 'user_registration >= ' . $range ) );
```

Comment:
Please pass ``__METHOD__`` as ``$fname`` argument to ease debugging of SQL queries.